packages/application/package.json

```
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "@jupyterlab/application": "^3.2.0",
-    "@jupyterlab/coreutils": "^5.2.0",
+    "@jupyterlab/coreutils": "^3.2.0",
```

Looks like an unintentional change? I think it should stay on `^5.2.0`:

```suggestion
    "@jupyterlab/coreutils": "^5.2.0",
```